src/erlcloud_sns.erl

```diff
@@ -367,11 +369,11 @@ list_subscriptions_by_topic_all(TopicArn, Config) ->
 list_platform_applications() ->
     list_platform_applications(undefined).
 
--spec list_platform_applications/1 :: (undefined|string()) -> [sns_application()].
+-spec list_platform_applications/1 :: (undefined|parameter()) -> [sns_application()].
 list_platform_applications(NextToken) ->
     list_platform_applications(NextToken, default_config()).
 
--spec list_platform_applications/2 :: (undefined|string(), aws_config()) -> [sns_application()].
+-spec list_platform_applications/2 :: (undefined|parameter(), aws_config()) -> [sns_application()].
```

Comment:
Line 376 is too long: -spec list_platform_applications/2 :: (undefined|parameter(), aws_config()) -> [sns_application()]..